staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -659,6 +659,11 @@ func (g *Cloud) Instances() (cloudprovider.Instances, bool) {
 	return g, true
 }
 
+// InstancesV2 returns an implementation of InstancesV2 for Google Compute Engine.
+func (g *Cloud) InstancesV2() (cloudprovider.InstancesV2, bool) {
+	return g, true
```

Comment:
set this to false and let GCE OWNERs flip this on?

Reply:
Set as false, while I thought @wojtek-t might want gce use InstancesV2() to reduce api calls, right?

Reply:
We can do that when we will be migrating the controller. So I'm fine with leaving false for now.